extensions/amp-analytics/0.1/vendors/parsely.js

```diff
@@ -17,20 +17,26 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const PARSELY_CONFIG = jsonLiteral({
+  'vars': {
```

Comment:
Do you still need to add data here? This is just the generic template for parsely integration so it will be sent for every customer who doesn't have a custom implementation. Customers can choose to override these variables, `requests`, `pageview`, and `heartbeat` in [their amp configuration in their implementation](https://www.parse.ly/help/integration/google-amp).

Reply:
I'm not sure it's strictly necessary, but the intent here was to make the default AMP requests mimic the behavior of the standard tracker. With the standard Parse.ly tracker, the `data` property is always sent on all events, and it's default value is an empty object (`{}`) in cases where integrators don't pass any custom data. Without this variable definition, the AMP tracker equivalent would send an empty string instead. I figured consistency would make it easier to reason about in downstream systems, but if you think that adds too much overhead to the default template, we can remove.